connect-example/dependencies/dependencies.txt

```diff
@@ -305,6 +305,7 @@
 |    |    |    |    +--- androidx.core:core:1.0.0 -> 1.13.0 (*)
 |    |    |    |    \--- androidx.customview:customview:1.0.0 (*)
 |    |    |    +--- org.jetbrains.kotlin:kotlin-stdlib:1.8.22 -> 2.0.21 (*)
+|    |    |    +--- androidx.fragment:fragment-compose:1.8.4 (c)
```

Comment:
Do you know where this compose version is coming from? We are using older versions in the rest of the SDK (1.6.8 generally and I think 1.7.4 in paymentsheet-example)

Reply:
This is the Androidx fragment library. It's actually not related to the compose version used in the SDK, and matches the other versions of fragments in use elsewhere: https://developer.android.com/jetpack/androidx/releases/fragment#1.8.4

Reply:
ah gotcha, thanks for the context!